internal/controllers/requirements/git/git_mock.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package git
+
+import (
+	mockutils "github.com/ZupIT/horusec-devkit/pkg/utils/mock"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockGit struct {
+	mock.Mock
+}
+
+func NewRequirementsGitMock() *MockGit {
+	return new(MockGit)
+}
+
+func (m *MockGit) ValidateGit() error {
+	args := m.MethodCalled("ValidateGit")
+	return mockutils.ReturnNilOrError(args, 0)
+}
```

Comment:
Mock implementations should be create on `testutil` package.

Reply:
i thought about that, but in this case the only code that would be using those mocks is the requirement test, other ones would be using requirements mock, i don't think there is a need to move to testutils if it won't  be used by any other package

Reply:
I think that all testing stuffs should be created on `testutil` package, even if just one test use, I don't think that main packages with test helpers is a good approach. 

All other mocks implementations is implemented there, I think that would be confusing some mocks implemented on `testutil` and others not.